README.md

```diff
@@ -17,11 +32,438 @@ Xcode 16 or later.
+- Chat room occupancy, i.e total number of connections and presence members.
+- Typing indicators
+- Room-level reactions (ephemeral at this stage)
+
+If there are other features you'd like us to prioritize, please [let us know](https://forms.gle/mBw9M53NYuCBLFpMA).
+
+## Usage
+
+You will need the following prerequisites:
+
+- An Ably account
+    - You can [sign up](https://ably.com/signup) to the generous free tier.
+
+- An Ably API key
+    - Use the default or create a new API key in an app within
+      your [Ably account dashboard](https://ably.com/dashboard).
+    - Make sure your API key has the
+      following [capabilities](https://ably.com/docs/auth/capabilities): `publish`, `subscribe`, `presence`, `history` and
+      `channel-metadata`.
+
+To instantiate the Chat SDK, create an [Ably client](https://ably.com/docs/getting-started/setup) and pass it into the Chat constructor:
+
+```swift
+let realtimeOptions = ARTClientOptions()
+realtimeOptions.key = "<API_KEY>"
+realtimeOptions.clientId = "<clientId>"
+let realtime = ARTRealtime(options: realtimeOptions)
+let chatClient = DefaultChatClient(realtime: realtime, clientOptions: nil)
+```
+
+You can use [basic authentication](https://ably.com/docs/auth/basic) i.e. the API Key directly for testing purposes,
+however it is strongly recommended that you use [token authentication](https://ably.com/docs/auth/token) in production
+environments.
+
+To use Chat you must also set a [`clientId`](https://ably.com/docs/auth/identified-clients) so that users are
+identifiable.
+
+## Connections
+
+The Chat SDK uses a single connection to Ably, which is exposed via the `ChatClient#connection` property. You can use this
+property to observe the connection state and take action accordingly.
+
+### Current connection status
+
+You can view the current connection status at any time:
+
+```swift
+let status = await chatClient.connection.status
+let error = await chatClient.connection.error
+```
+
+### Subscribing to connection status changes
+
+You can subscribe to connection status changes by registering a listener, like so:
+
+```swift
+let subscription = chatClient.connection.onStatusChange(bufferingPolicy: .unbounded)
+for await statusChange in subscription {
+    print("Connection status changed to: \(statusChange.current)")
+}
+```
+
+To stop listening to changes, call the `finish` method on the returned subscription instance:
+
+```swift
+subscription.finish()
+```
+
+## Chat rooms
+
+### Creating or retrieving a chat room
+
+You can create or retrieve a chat room with name `"basketball-stream"` this way:
+
+```swift
+let room = try await chatClient.rooms.get(roomID: "basketball-stream", options: RoomOptions())
+```
+
+The second argument to `rooms.get` is a `RoomOptions` argument, which tells the Chat SDK what features you would like your room to use and
+how they should be configured.
+
+You can also use `RoomOptions()` to enable all room features with the default configuration.
```

Comment:
That's not true — they need to use `.allFeaturesEnabled`.

Reply:
Updated